docs/user-guide/connectors/custom-connectors.rst

```diff
@@ -31,6 +31,17 @@ To send a message, you would run a command like:
 
 where ``myio`` is the name of your component.
 
+You can also send extra information from your front-end to your
+action server, with it you can use, e.g. name, locations, or tokens
+in our actions. To send a message with ``metadata`` you would run
+a command like:
+
+.. code-block:: bash
+
+    curl -XPOST http://localhost:5000/webhooks/myio/webhook \
+      -d '{"sender": "user1", "message": "here is my token: <token>"}, "metadata: {"type": "token"}' \
+      -H "Content-type: application/json"
+
```

Comment:
@erohmensing How about including the `UserMessage` class definition in here? Same for `InputChannel` and `OutputChannel` (that's probably out of scope of this pr)

Reply:
Good idea 👍 that would show where "sender" "message" and "metadata" are coming from

Reply:
@arthurTemporim Doing the `autoclass` stuff for `UserMessage` would be enough here imo.